### ISSUE TITLE
Not working with the latest version

### ISSUE THREAD

**Issue by tarkanlar** [clappr/clappr-level-selector-plugin#4]
Hi,
This works awesome with 0.1.3 version of Clappr

but I can't switch quality with  the  version 0.1.5 i just see auto button

I tried with the  version 0.1.6 as well but I got black screen and this error on console log

```js
Uncaught TypeError: this.el.globoPlayerSetflushLiveURLCache is not a function
```

**Comment by towerz** [comment]
@tarkanlar This is fixed in 0.0.7. Please check if this works for you.

**Comment by tarkanlar** [comment]
Hey @towerz works awesome thanks
